api/test/integration/test_webhook_endpoints.tavern.yaml

```diff
@@ -0,0 +1,174 @@
+---
+test_name: POST /events
+
+stages:
+
+  - name: Send bulk with one element
+    request: &webhook_request
+      verify: false
+      url: "{protocol:s}://{host:s}:{port:d}/events"
+      method: POST
+      headers:
+        Authorization: "Bearer {test_login_token}"
+      json:
+        events:
+          - foo event
+    response:
+      status_code: 200
+      json:
+        data:
+          affected_items:
+            - foo event
+          total_affected_items: 1
+          total_failed_items: 0
+          failed_items: []
+        message: All events were forwarded to analisysd
+        error: 0
+
+  - name: POST /events with multiple elements
```

Comment:
The webhook name is not usually included in the test name. For consistency with other tests, this is the expected format:
- `Send multiple events in one request` -> If no errors are expected.
- `Try to send multiple events in one request` -> If an error or all failed items are expected.

In this case, `Send multiple events in one request` should be used.